xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/server/ServerInfo.java

```diff
@@ -192,18 +192,18 @@ default boolean isLedgerInCompleteLedgers(final UnsignedLong ledgerIndex) {
    * individual server's load factor, the cluster's load factor, the open ledger cost and the overall network's load
    * factor.
    *
-   * @return An {@link UnsignedInteger} representing the load factor.
+   * @return An optionally-present {@link Double} representing the load factor.
    */
   @JsonProperty("load_factor")
-  UnsignedInteger loadFactor();
+  Optional<Double> loadFactor();
```

Comment:
I'm coming late to this party, but I'd like us to reconsider the use of `Double`. My first inclination was BigDecimal on grounds of precision/resolution/etc, but haven't looked too deeply into the XRPL.org docs. Thoughts?

Reply:
The xrpl.org docs are somewhat unclear, they just say this is a "Number".  To be safe I think `BigDecimal` is probably the way to go.

Reply:
I agree that BigDecimal is better for this